app/src/main/java/io/github/droidkaigi/confsched2018/presentation/sessions/feedback/SessionsFeedbackViewModel.kt

```diff
@@ -0,0 +1,70 @@
+package io.github.droidkaigi.confsched2018.presentation.sessions.feedback
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import android.arch.lifecycle.ViewModel
+import io.github.droidkaigi.confsched2018.data.repository.SessionRepository
+import io.github.droidkaigi.confsched2018.model.SessionFeedback
+import io.github.droidkaigi.confsched2018.presentation.Result
+import io.github.droidkaigi.confsched2018.presentation.common.mapper.toResult
+import io.github.droidkaigi.confsched2018.util.rx.SchedulerProvider
+import io.reactivex.Observable
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.addTo
+import javax.inject.Inject
+
+class SessionsFeedbackViewModel @Inject constructor(
+        private val repository: SessionRepository,
+        private val schedulerProvider: SchedulerProvider
+) : ViewModel() {
+
+    var sessionId: String = ""
+    var sessionTitle: String = ""
+
+    private var mutableSessionFeedback = MutableLiveData<Result<SessionFeedback>>()
+    var sessionFeedback: LiveData<Result<SessionFeedback>> = mutableSessionFeedback
+
+    private val compositeDisposable: CompositeDisposable = CompositeDisposable()
+
+    fun init() {
```

Comment:
(You can merge this without fixing it.)
Please implement LifecycleObserver and ` @OnLifecycleEvent(Lifecycle.Event.ON_CREATE)`

Reply:
I merge this. and create issue to applay lifecycle event 🙇